daml-lf/engine/src/test/scala/com/digitalasset/daml/lf/engine/MinVersionTest.scala

```diff
@@ -96,27 +96,29 @@ final class MinVersionTest
                 commandId = UUID.randomUUID.toString,
                 party = party,
                 commands = Seq(
-                  Command().withCreate(
-                    CreateCommand(
-                      templateId = Some(
-                        Identifier(
-                          packageId = dar.main._1,
-                          moduleName = "Iou",
-                          entityName = "Iou",
-                        )
-                      ),
-                      createArguments = Some(
-                        Record(
-                          None,
-                          Seq(
-                            RecordField("issuer", Some(Value().withParty(party))),
-                            RecordField("owner", Some(Value().withParty(party))),
-                            RecordField("currency", Some(Value().withText("EUR"))),
-                            RecordField("amount", Some(Value().withNumeric("10.0"))),
-                            RecordField("observers", Some(Value().withList(List()))),
-                          ),
-                        )
-                      ),
+                  Command.of(
```

Comment:
seems like an unnecessary change which doesn’t really simplify things.

Reply:
Sorry, went a bit overboard; it was just to make IntelliJ stop complaining. I shall revert it.

Reply:
There was an issue with using a constructor for ScalaPB generated classes. I believe the same issue exists for `with`. @SamirTalwar-DA, do you recall it?

Reply:
No, not from memory, and this clearly worked before, so I have reverted it.